.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         with:
           node-version: ${{ matrix.node_version }}
       - name: Install
-        run: yarn install --pure-lockfile
+        run: yarn install --pure-lockfile --ignore-engines
```

Comment:
Had to add `--ignore-engines` since some of the new dependencies require more modern Node.js versions, preventing dependencies from being installed.